BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -29,148 +29,157 @@ class SideMenuFilterViewController: UITableViewController {
     @IBOutlet weak var cancelButton: UIButton!
     @IBOutlet weak var contentViewCell: UIView!
     @IBOutlet weak var slideValueLabel: UILabel!
-    
-    var distanceFilter = 0
+
+    var distanceFilter: CGFloat = 0.00
     var isRestaurant = false
     var isStore = false
     var isOpenNow = false
     var delegate: SideMenuFilterDelegate?
     
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         styleMenu()
+
+    }
+
+    func passingDistance() -> CGFloat {
         
+        return distanceFilter
+    
     }
     
     @IBAction func sliderDidSlide(_ sender: UISlider) {
+
+        distanceFilter = CGFloat(slider.value)
         
-        distanceFilter = Int(slider.value)
+        let roundedNumber = Double(round(distanceFilter * 100)/100)
```

Comment:
Might just be simpler to cast it to an `Int` and then back, instead of doing more cumbersome mathematics